src/chapters/css/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
+
+For the exercises, add the following style rules to ``style.css``:
+
+1. Change the background color to yellow.
+2. Change all paragraph text color to red.
```

Comment:
OUCH! Red text on a yellow background. My eyes are hurting just thinking about this. ;-)

Reply:
Very true! Maybe I will make it green on yellow.